src/main/scala/com/fulcrumgenomics/bam/FilterBam.scala

```diff
@@ -98,6 +101,7 @@ class FilterBam
           minMappedBases.exists(count => countMappedBases(rec) < count)
 
         if (throwOut) {
+          rejects.foreach { writer => writer += rec }
```

Comment:
This reads a little easier:

```suggestion
          rejects.foreach(writer.write)
```